packages/xpetra/sup/Utils/Xpetra_MatrixMatrix.hpp

```diff
@@ -152,7 +150,7 @@ namespace Xpetra {
             *Bprime, false, beta,
             C);
         return rcp(new CrsWrap(rcp_implicit_cast<CrsType>(rcp(new XTCrsType(C)))));
```

Comment:
This line of code is so weird, but you didn't write it :-)

Reply:
@mhoemmen Nope, that's just how Xpetra works :)
I did look all over the place for a helper function that does this (Tpetra::CrsMatrix to Xpetra::Matrix) but I don't think there is one. The inverse exists though (probably because it has a bunch of dynamic_casts that need to be checked at each step).

Reply:
:octopus: